app/controllers/api/devices_controller.rb

```diff
@@ -5,7 +5,9 @@ class DevicesController < Api::AbstractController
 
     # GET /api/device
     def show
-      render json: current_device
+      current_device
+        .if_null { render json: {error: "add device to account"}, status: 404 }
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.